assets/javascripts/running.js

```diff
@@ -262,6 +262,21 @@ function initLivelog() {
 
 /********* LIVE LOG END *********/
 
+/********* LIVE TERMINAL *********/
+
+function initLiveterminal() {
+    // start stream
+    var livelog = $('#liveterminal');
+    var events = new EventSource(livelog.data('url'));
+    events.addEventListener('message', function(event) {
+        livelog[0].innerHTML += JSON.parse(event.data)[0];
+        if (scrolldown) {
+            livelog[0].scrollTop = livelog[0].scrollHeight;
+        }
+    }, false);
+}
```

Comment:
this looks pretty much similar to the previous function initLivelog(). Please 
[DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself).